<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BankAssignmentMatrix</class>
 <widget class="QDialog" name="BankAssignmentMatrix">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bank Assignment Matrix</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnGetChildren">
       <property name="text">
        <string>Get Children</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddRow">
       <property name="text">
        <string>Add Row</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRemoveRow">
       <property name="text">
        <string>Remove Row</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddColumn">
       <property name="text">
        <string>Add Column</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRemoveColumn">
       <property name="text">
        <string>Remove Column</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tblMatrix">
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>4</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>5</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>3</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnCreateBanks">
       <property name="text">
        <string>Create Banks</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAssignMedia">
       <property name="text">
        <string>Assign Media</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbxSingleSoundOnly">
       <property name="text">
        <string>Single Sound Only</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
